README.md

```diff
@@ -69,6 +69,7 @@ return {
 | Command              | Description                                                                 |
 | -------------------- | --------------------------------------------------------------------------- |
 | `CdProject`          | change working directory                                                    |
+| `CdProjectForWindow`          | change working directory for current window (:h lcd for more)                                             |
```

Comment:
I don't want to add a new cmd here; I think it should be great to have it as a config option for the `CdProject` cmd